deploy/acm/policies/patch-policies-status.sh

```diff
@@ -20,3 +20,8 @@ curl -k -s -X PATCH -H "Accept: application/json, */*" \
 http://localhost:8001/apis/policy.open-cluster-management.io/v1/namespaces/$4/policies/policy4-common-sriov-sub-policy/status \
 --data '{"status":{"compliant":"NonCompliant","status":[{"clustername":"spoke4","clusternamespace":"spoke4","compliant":"NonCompliant"}, {"clustername":"spoke5","clusternamespace":"spoke5","compliant":"NonCompliant"}, {"clustername":"spoke6","clusternamespace":"spoke6","compliant":"NonCompliant"}]}}'
 
+curl -k -s -X PATCH -H "Accept: application/json, */*" \
+-H "Content-Type: application/merge-patch+json" \
+http://localhost:8001/apis/policy.open-cluster-management.io/v1/namespaces/$4/policies/policy5-subscriptions/status \
+--data '{"status":{"compliant":"NonCompliant","status":[{"clustername":"spoke4","clusternamespace":"spoke4","compliant":"NonCompliant"}, {"clustername":"spoke5","clusternamespace":"spoke5","compliant":"NonCompliant"}, {"clustername":"spoke6","clusternamespace":"spoke6","compliant":"NonCompliant"}]}}'
+
```

Comment:
Is this test code, or used in production?

Reply:
Just for testing.